frontend/src/scenes/surveys/SurveyNoResponsesBanner.tsx

```diff
@@ -0,0 +1,19 @@
+import { SurprisedHog } from 'lib/components/hedgehogs'
```

Comment:
new empty state for survey without responses. not used atm but will be in the new questions:

![image](https://github.com/user-attachments/assets/0e6e0bf3-acaa-452c-8661-db603a78d7e7)